python/torch_mlir/dynamo.py

```diff
@@ -62,7 +62,6 @@ def _get_decomposition_table():
         aten.native_group_norm_backward,
         aten.sigmoid_backward,
         aten._native_batch_norm_legit,
-        aten._native_batch_norm_legit_no_training,
```

Comment:
Hmm so we are disabling a nightly feature since it is broken on stable? Shouldn't this be guarded for stable ?